pkg/workflows/create.go

```diff
@@ -359,9 +371,49 @@ func (s *DeleteBootstrapClusterTask) Run(ctx context.Context, commandContext *ta
 	if commandContext.OriginalError == nil {
 		logger.MarkSuccess("Cluster created!")
 	}
-	return nil
+	return &InstallPackageControllerTask{}
 }
 
 func (s *DeleteBootstrapClusterTask) Name() string {
 	return "delete-kind-cluster"
 }
+
+func (s *InstallPackageControllerTask) Run(ctx context.Context, commandContext *task.CommandContext) task.Task {
+	logger.Info("Installing curated packages controller on workload cluster")
+	kubeConfig := kubeconfig.FromClusterName(commandContext.ClusterSpec.Cluster.Name)
+	deps, err := curatedpackages.NewDependenciesForPackages(ctx, kubeConfig)
+	if err != nil {
+		logger.MarkFail("Error when installing curated packages on workload cluster; please install through eksctl anywhere install packagecontroller command", "error", err)
+		return nil
+	}
+	chart := commandContext.ClusterSpec.VersionsBundle.VersionsBundle.PackageController.HelmChart
+	pc := curatedpackages.NewPackageControllerClient(deps.Helm, deps.Kubectl, kubeConfig, chart.Image(), chart.Name, chart.Tag())
+	err = pc.InstallController(ctx)
+	if err != nil {
+		logger.MarkFail("Error when installing curated packages controller on workload cluster; please install through eksctl anywhere install packagecontroller command", "error", err)
+		return nil
+	}
+	return nil
+}
+
+func (s *InstallPackageControllerTask) Name() string {
+	return "install-package-controller"
+}
+
+func installCuratedPackages(ctx context.Context, clusterName string, packagesLocation string) {
+	kubeConfig := kubeconfig.FromClusterName(clusterName)
+	deps, err := curatedpackages.NewDependenciesForPackages(ctx, kubeConfig)
+	if err != nil {
+		logger.MarkFail("Error when installing curated packages on workload cluster; please install through eksctl anywhere create packages command", "error", err)
```

Comment:
All these messages are exactly the same, not sure if there is enough context in the error, so it might be handy to have a little difference in each case

Reply:
Fixed the repetition issue.